core/jvm/src/main/scala/fs2/interop/flow/package.scala

```diff
@@ -82,12 +82,13 @@ package object flow {
     *                   like requesting elements from a database.
     *                   The publisher can use this `bufferSize` to query elements in batch.
     *                   A high number will also lead to more elements in memory.
+    *                   By default uses [[defaultBufferSize]].
     * @param subscribe The effectual function that will be used to initiate the consumption process,
     *                  it receives a [[Subscriber]] that should be used to subscribe to a [[Publisher]].
     *                  The `subscribe` operation must be called exactly once.
     */
   def fromPublisher[F[_], A](
-      bufferSize: Int
+      bufferSize: Int = defaultBufferSize
   )(
       subscribe: Subscriber[A] => F[Unit]
   )(implicit
```

Comment:
Even with the default this will require an empty set of parens e.g.
```scala
fromPublisher()(subscriber => ...)
```
not the end of the world though.

Reply:
IMHO, that is a good idea, one should not rely on the default, and making it look weird is a good way to motivate folks to use an appropriate value.

Reply:
If the goal is to discourage use of the default, why have it all then 😅 but fair enough, I don't feel strongly any way about this.

We could also expose `defaultBufferSize` as a method on `flow`, just like in the JDK. So users can use that one if they wish.

Reply:
Sorry, I didn't express myself clearly.
My point is mostly that users should be conscious of that `bufferSize` parameter.